easybuild/easyconfigs/d/dune-core/dune-core-2.8.0.post1-foss-2020b.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'PythonBundle'
+
+name = 'dune-core'
+version = '2.8.0.post1'
+
+homepage = "https://www.dune-project.org/groups/core/"
+description = """The Dune core modules build the stable basis of Dune. 
+They follow a consistent release cycle and have high requirements regarding stability and backwards compatibility. 
+These modules build the foundation for higher-level components.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('CMake', '3.18.4'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+builddependencies = [
```

Comment:
please keep the builddependencies and dependencies together in the easyconfig file

Reply:
I re-arranged the code, based on how things are ordered in SciPy-bundle-2021.10-foss-2021b.eb